lightly/models/modules/heads.py

```diff
@@ -0,0 +1,117 @@
+# TODO
+
+from typing import Union, List, Tuple
+
+import torch
+import torch.nn as nn
+from torch.nn.modules.batchnorm import BatchNorm1d
+
+
+class ProjectionHead(nn.Module):
+    """TODO
+    
+    """
+
+    def __init__(self, layers: List[Tuple[int, int, nn.Module, nn.Module]]):
+
+        self.ffnn = []
```

Comment:
would give it a more meaningful variable name. E.g. `self.block`, `self.head`, `self.linear`?

Reply:
I agree, I don't know what ffnn stands for. Why not `self.layers`?

Reply:
feed forward neural network... but I'll change it 😄